Updating CI/CD scripts to adapt recent version + addition of new NGSI-LD test cases

Aniket-NEC opened the pull request:
Pull request contains below addition:

- Updated Persistence Test-cases to adapt to new version, so it corrects the failing build.
- Addition of new test cases based on Geo-location, visualization feature and NGSI-LD use cases.
- Path for storage of internal metadata kept same as public/data/meta

showersky reviewed (changes requested):
did you make any change to designer/main.js? Please fetch the latest version and merge into your feature branch and then make a new PR.

Aniket-NEC commented:
@showersky I have updated the pull request with the latest version of development branch. 

I was facing this issue in designer/main.js : 
```
node:internal/process/promises:265
            triggerUncaughtException(err, true /* fromPromise */);
            ^

[Error: ENOENT: no such file or directory, open 'public/data/meta/.db.json.tmp'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: 'public/data/meta/.db.json.tmp'
}
```
I faced this issue while building components from source. As in [.github/workflows](https://github.com/smartfog/fogflow/blob/development/.github/workflows/main.yml), the components are built from source, the persistence testcases are failing. 

So, I have added a new line of code(in designer/main.js) to make a working directory for the storage of internal metadata.
`fs.mkdir('public/data/meta/', { recursive: true })`

showersky commented:
this folder will be mapped from a local path into the container when we deploy the designer. But it is good to add this so that the folder can be created if it does not exist. I have added it into the development branch. 
Could you pull the new development branch and then make a new PR for the other test scripts?
